src/mixins/CanvasTools.ts

```diff
@@ -0,0 +1,77 @@
+import Konva from 'konva'
```

Comment:
what has changed in this file? or have you just moved it?

Reply:
it's a class now, rather than an object. See https://github.com/vuejs/vue-class-component#using-mixins